WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorView.swift

```diff
@@ -83,20 +83,30 @@ struct ProductSelectorView: View {
 
     var body: some View {
         VStack(spacing: 0) {
-            SearchHeader(text: $viewModel.searchTerm, placeholder: Localization.searchPlaceholder, onEditingChanged: { isEditing in
-                searchHeaderisBeingEdited = isEditing
-            })
+            HStack {
+                SearchHeader(text: $viewModel.searchTerm, placeholder: Localization.searchPlaceholder, onEditingChanged: { isEditing in
+                    searchHeaderisBeingEdited = isEditing
+                })
                 .padding(.horizontal, insets: safeAreaInsets)
                 .accessibilityIdentifier("product-selector-search-bar")
-            Picker(selection: $viewModel.productSearchFilter, label: EmptyView()) {
-                ForEach(ProductSearchFilter.allCases, id: \.self) { option in
-                    Text(option.title)
-                            }
-                        }
-                        .pickerStyle(.segmented)
-                        .padding(.leading)
-                        .padding(.trailing)
-                        .renderedIf(searchHeaderisBeingEdited)
+                Picker(selection: $viewModel.productSearchFilter, label: EmptyView()) {
+                    ForEach(ProductSearchFilter.allCases, id: \.self) { option in Text(option.title) }
+                }
+                .pickerStyle(.segmented)
+                .padding(.leading)
+                .padding(.trailing)
+                .renderedIf(searchHeaderisBeingEdited)
+                Button(action: {
+                    // TODO: gh-12011
```

Comment:
is this left to be implemented here or?

Reply:
Sorry, I should have written the full link 😅  Yes, will be implemented separately on issue 12011: https://github.com/woocommerce/woocommerce-ios/issues/12011